hatchways/team-bluejay
# Be orders model

To test:

Migrate DB

Go to checkout with no items in cart. You should not be able to checkout.

Add items to cart, proceed with payment. Using stripe test card details
[https://stripe.com/docs/testing](url)

cart should be empty. 

Backend should be populated with your order.
